fdbserver/EncryptKeyProxy.actor.cpp

```diff
@@ -862,9 +960,13 @@ ACTOR Future<Void> refreshBlobMetadataCore(Reference<EncryptKeyProxyData> ekpPro
 		}
 
 		ekpProxyData->blobMetadataRefreshed += rep.metadataDetails.size();
-
+		ekpProxyData->kmsConnectorHealthStatus = KMSHealthStatus{ true, true, now() };
```

Comment:
we still want to check if response `vector.size > 1` before updating the KmsHealthStatus?